akka-http/src/main/scala/scala/util/matching/JavaConverters.scala

```diff
@@ -0,0 +1,10 @@
+package scala.util.matching
```

Comment:
we can't cross proper package boundaries :-) Will have to live in akka.http.something.something :) AFAIR we (sadly) so have a util class somewhere, you could put it there)